src/Documentation/implementation/streams_implementation/azure_queue_streams.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Azure Queue Streams Implementation Details
+---
+
+# Orleans Azure Queue Streams Implementation Details
+
+Each stream provider (Azure Queues, EventHub, SMS, SQS, ...) has it's own queue specific details and configuration.
+This section provides some details about the usage, configuration and implementation of **Orleans Azure Queue Streams**.
+More details are available in the streaming tests, which contain most of the configuration options, specifically [**`AQClientStreamTests`**](https://github.com/dotnet/orleans/tree/master/test/Extensions/TesterAzureUtils/Streaming/AQClientStreamTests.cs), [**`AQSubscriptionMultiplicityTests`**](https://github.com/dotnet/orleans/tree/master/test/Extensions/TesterAzureUtils/Streaming/AQSubscriptionMultiplicityTests.cs), and the extension functions for [**`IAzureQueueStreamConfigurator`**](https://github.com/dotnet/orleans/tree/master/src/Azure/Orleans.Streaming.AzureStorage/Providers/Streams/AzureQueue/AzureQueueStreamBuilder.cs)  and [**`ISiloPersistentStreamConfigurator`**](https://github.com/dotnet/orleans/tree/master/src/Orleans.Runtime.Abstractions/Streams/ISiloPersistentStreamConfigurator.cs).
+
+## Overview
+
+Orleans Azure Queue requires the package **Microsoft.Orleans.Streaming.AzureStorage** and the silo configuration at startup allow to specify the connection string
+
+``` csharp
+hostBuilder
+  .AddAzureQueueStreams<AzureQueueDataAdapterV2>("AzureQueueProvider", optionsBuilder => optionsBuilder.Configure(options => {
+    options.ConnectionString = "xxx";
+  }))
+  // also a PubSubStore is needed, as example using Azure Table Storage
```

Comment:
Pubsub needs only be configured using default or explicit subscriptions.  It's not needed if using only implicit subscriptions only.  Not sure how much we want to get into subscription configuration in this section, but maybe add a 'by default' qualifier to the the comment?

Reply:
Is the comment better now?
Interesting, I believed it was needed by design to store the subscriptions. I looked to the documentation by I haven't found mention that this is "optional". I don't think this page is the right place to dig in this concept, as it is a more global.

Reply:
If this is something that you want to be covered in an other page, I can try to submit another PR.